front/style/_forms.scss

```diff
@@ -61,3 +61,24 @@
         }
     }
 }
+
+.login {
+    &--box {
+        margin: 20px auto 0;
+        background-color: white;
+
+        &--error {}
```

Comment:
then, they will add it. we're not supposed to put empy css code in our css files, just a a markers :) Integrators will find them